diagnostic.md

```diff
@@ -36,19 +36,23 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+the path is absolute because it starts with /Users, indicatiing that it is a
+unique file path to the users' home directory.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+we would use a relative path to refer to the image. it is not stored in a
```

Comment:
It would be an absolute path. You need the root (url) in order to access the file on a server somewhere.